cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js

```diff
@@ -46,6 +46,7 @@ function FileActionsDropdown({ file, table_type, setOpenModal, setSelectedFile,
     }, 0);
     return () => clearTimeout(timeout);
   }, []);
+  const isOnline = Object.keys(file.availability).length === 1 && file.availability.hasOwnProperty('online');
```

Comment:
if a file has both hot and cold copies it is not marked as online?

Reply:
Yes, if a file has two copies (and one of them is online), then the file is online